pages/dmp_supporting_info/support_08_preserve_publish.md

```diff
@@ -63,67 +63,72 @@ When deciding where to archive your data, make suere you chose an archive where
+Archiving of datasets includes the deposition of datasets in a research data archive, openly or with restricted access, and equipped with a persistent identifier (PID) as well as data that should be preserved but not published. As mentioned above, research data should be made available at an early stage, following the principles "as open as possible, as closed as necessary". Funders and research institutions may specify more exact guidelines for this. E.g. the guidelines from the Research Council of Norway says that data underlying scientific articles should be made accessible latest at publication. Other research data should be made accessible within a reasonable amount of time, and never later than three years after the project has ended [(Policy for open access to research data, 2017)](https://www.forskningsradet.no/contentassets/e4cd6d2c23cf49d4989bb10c5eea087a/the-research-council-of-norways-policy-for-open-access-to-research-data.pdf).
 
-Several sub-questions allow to refer to information in other DMP chapters or external databases. Alternatively, it is always possible to enter a free-text answer.
+Several of the sub-questions here let you refer to information in other chapters of your DMP or in external databases. Alternatively, it is always possible to enter a free-text answer.
 
 #### Which metadata and documentation accompanies the dataset?
-Explain to the best of your knoledge what metadata (what metadata standard) is used for your dataset.
+Explain to the best of your knowledge what metadata (what metadata standard) is used for your dataset. Here you may reuse information from the metadata standard question in the 'Data documentation' chapter.
+
 Metadata is structured information about a dataset (or a document). _How to_ structure the information is done in various ways. Different metadata schemas are designed to capture the information, and such a schema is called a metadata standard. A metadata standard may be designed to suit a subject domain or a specific research area. There are, naturally, different information needs to describe a dataset with biological data, compared to a dataset which revolves on social sciences.
 A metadata schema may capture information by the use of controlled vocabularies. That way, information will be standardised and not be disrupted by different persons using different words or typology for same issues. Such a standardisation will preferably make use of ontologies from a subject domain, and metadata standards will thus often be subject specific. The use of controlled vocabularies and ontologies will increase the quality and findability of the archived data.
```

Comment:
```suggestion
A metadata schema may capture information by the use of controlled vocabularies to avoid ambiguity. That way, information will be standardised and not be disrupted by different persons using different words or typology for same issues. Such a standardisation will preferably make use of ontologies from a subject domain, avoiding misunderstandings if a term is used differently in different disciplines. The use of controlled vocabularies and ontologies will increase the quality and findability of the archived data.
```